editor/editor_node.h

```diff
@@ -878,6 +878,8 @@ class EditorNode : public Node {
 
 	Control *get_gui_base() { return gui_base; }
 
+	void save_edited_scene_no_error_dialogs();
```

Comment:
The name is a bit misleading. It save the scene only if saving it would not lead to errors, otherwise it skips saving it.
Something like `save_scene_if_valid()` would be more fitting.